pyverdict/pyverdict/postgresconverter.py

```diff
@@ -0,0 +1,227 @@
+import psycopg2.tz
```

Comment:
Do we need this? If so, this module should also be in the dependency list of setup.py.

Reply:
pscopg2 returns time related results with an object of type FixedOffsetTimezone as the tzinfo attribute.  To ensure equality between our results and psycopg2 I returned the same timezone object as they do.  However, I could change this to instead take a more standard tzinfo value and check for equality differently in the tests.  For now I've added psycopg2 as a dependency in setup.py, though.  I can change this if useful to be handled in equality checking of the tests if useful.

Reply:
Let's return something standard e.g., Python's datetime.datetime, rather than the special type. For testing, we can simply compare the content, not the types. Lastly, please remove the pscopg2 from the dependency.